packages/fluent-theme/src/private/VariantComposer.ts

```diff
@@ -0,0 +1,27 @@
+import { createContext, memo } from 'react';
+import createVariantComposer from './createVariantComposer';
+
+type VariantName = 'fluent' | 'copilot' | '';
+
+export type VariantList = `${VariantName}`;
```

Comment:
Why not directly `export type VariantList = 'fluent' | 'copilot' | ''`?

Is `'fluent' === ''`? If yes, should we keep it just `'copilot' | ''`?

Reply:
It may be useful to not include any variant e.g. to use Web Chat style options. The variant is defaulted to `fluent` with the ability to override to empty.

Reply:
Composing is nice, I am worry about the complexity later on.

Say, 1:1 chat vs. 1:N chat. Bot chat vs. human chat...

Reply:
I see it could be useful for example for dark variants where we need to adjust actual styles. As you may know we can't rely on media for fluent work unfortunately We can discuss this later when the need comes. I'm ok with changing type to whatever. Just API-wise it seems a good hint that the type may be not a single value at some point.

Reply:
Let's update it later, we have our directions in our head so this is good.